src/app/shared/components/timer/timer.component.html

```diff
@@ -0,0 +1,3 @@
+<p class="timer timer-title">
+  {{ title }}: <span class="timer timer-time">{{ time }}</span>
+</p>
```

Comment:
Suggestion for future improvements: it's more common to create a container with base class and put it's part in it
```
<div class='timer'>
<p class='timer-title'></p>
<span class='timer-time'></span>
</div>
```
so you don't have to duplicate _timer_ class twice.
Or you can try to style app-timer component using :host() selector as you do for timer)